hack/glide-update.sh

```diff
@@ -3,4 +3,22 @@
 glide update
 
 # need to remove the vendor tree from our types to be able to build against the "right" apimachinery and such for now
-glide install --strip-vendor github.com/openshift/origin
\ No newline at end of file
+glide install --strip-vendor github.com/openshift/origin
+
+dep init
+dep prune
+
+# we shouldn't have modified anything
+git status | grep modified | awk '{print $2}' | xargs git checkout
```

Comment:
You mean `git diff-index --name-only --diff-filter=M HEAD | xargs -r git checkout -f` ?